code/modules/clothing/glasses/glasses.dm

```diff
@@ -136,6 +136,13 @@
 	item_state = "glasses"
 	vision_correction = 1 //corrects nearsightedness
 
+/obj/item/clothing/glasses/regular/jamjar
+	name = "Jamjar Glasses"
+	desc = "Also known as Virginity Protectors."
+	icon_state = "jamjar_glasses"
+	item_state = "jamjar_glasses"
+	vision_correction = 1 //corrects nearsightedness

```

Comment:
glasses/regular has vision_correction set to 1, so anything with the path including that will also have vision_correction set to 1. look at https://github.com/tgstation/tgstation/pull/24373/files#diff-d7594f2da0715636602b06ec080ea828R146 , notice how it's still perscription glasses even though it does have a var for vision correction